Polyfills/Window/Source/Window.cpp

```diff
@@ -10,18 +11,23 @@ namespace Babylon::Polyfills::Internal
         constexpr auto JS_CLASS_NAME = "Window";
         constexpr auto JS_SET_TIMEOUT_NAME = "setTimeout";
         constexpr auto JS_A_TO_B_NAME = "atob";
+
         constexpr auto JS_ADD_EVENT_LISTENER_NAME = "addEventListener";
         constexpr auto JS_REMOVE_EVENT_LISTENER_NAME = "removeEventListener";
+        auto JS_DEVICE_PIXEL_RATIO_NAME = "devicePixelRatio";
     }
 
-    void Window::Initialize(Napi::Env env)
+    void Window::Initialize(Napi::Env env, void* windowPtr)
```

Comment:
We shouldn't need to pass the windowPtr to `Window::Intialize` because `Window` can just get it from `Graphics::Impl::GetNativeWindow()` right?

Reply:
Yeah, its legal for our polyfills to take dependencies on the core components, so I can make that change, and remove the member variable from the window class.

Reply:
We can either a) Link against Graphics and manage a reference to the external class, or b) link against GraphicsInternal and all its dependencies, and use Javascript to get a reference to the Graphics::Impl. I currently updated this PR to use a), as I've had some strange compilation issues with b), but I can take another look at this

Reply:
Had to tweak the Graphics cmake file, we can link against GraphicsInternal, and it cleans up our creation and makes it so we don't necessarily have to manage a reference to a Graphics object in the Window polyfill object. Definitely the better solution.